### ISSUE TITLE
Helpful if 'markupdeclaration' GitHub organization members were public

### ISSUE THREAD

**Issue by tgraham-antenna** [markupdeclaration/scratchpad#2]
If at least some of the members of the 'markupdeclaration' organization on GitHub made their membership public, then the organization would look more personable than the current anonymous state. (Anonymous until you look at who makes commits or approves PRs, that is.)

If a person's membership is public, then the 'markupdeclaration' logo will show up on their personal page on GitHub, which may increase the visibility of the organization.

**Comment by liamquin** [comment]
Hmm, i don't seem to be a Member. Who is, i wonder? I don't see such a logo on your profile page either.

**Comment by ndw** [comment]
I'll work on that. Mea culpa, I expect.

**Comment by tgraham-antenna** [comment]
It's an individual thing.  You can't change someone else's setting. (And, really, that's the way it should be.)

**Comment by ndw** [comment]
Right. I've created a 'members' team and invited you both. Let's see if this does what we want. Please join when you get the invites!

**Comment by crism** [comment]
“This organization has no public members. You must be a member to see who’s a part of this organization.” Individuals should be able to choose whether to be members, as @tgraham-antenna notes, but the membership should be public.

**Comment by ndw** [comment]
Ok. I found the knob to turn to make my membership public. And I invited you to join as well, @crism. First one to accept an invitation and tell me if you can set your visibility to public without my help wins a prize.

**Comment by crism** [comment]
Thanks, @ndw. I was able to set my membership to public on my own; in the list of members, privacy is displayed, and is a dropdown menu for oneself.

**Comment by ndw** [comment]
Fabulous. A free beer on me next time we're colocated. That is to say, I'll buy you a beer; that is not an invitation to pour one on me.
I'll add some prose to the contact page on markupdeclaration.org about joining the group.

**Comment by liamquin** [comment]
On Fri, 2018-09-21 at 14:08 -0700, Norman Walsh wrote:
> Ok. I found the knob to turn to make my membership public. And I
> invited you to join as well, @crism. First one to accept an
> invitation and tell me if you can set your visibility to public
> without my help wins a prize.

Haha i think Chris was first :) butyes, i can go public too.

Liam


> 
-- 
Liam Quin - web slave for https://www.fromoldbooks.org/
with fabulous vintage art and fascinating texts to read.
Click here to have the slave beaten.
